File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/commands/CreateConference.cs
using System;
using ENode.Commanding;

namespace ConferenceManagement.Commands
{
    public class CreateConference : Command<Guid>
    {
        public string AccessCode { get; set; }
        public string OwnerName { get; set; }
        public string OwnerEmail { get; set; }
        public string Slug { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Location { get; set; }
        public string Tagline { get; set; }
        public string TwitterSearch { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/management/commands/CreateConference.java
package com.microsoft.conference.common.management.commands;

import lombok.Getter;
import lombok.Setter;
import org.enodeframework.commanding.Command;

import java.util.Date;

@Getter
@Setter
public class CreateConference extends Command<String> {
    private String accessCode;
    private String ownerName;
    private String ownerEmail;
    private String slug;
    private String name;
    private String description;
    private String location;
    private String tagline;
    private String twitterSearch;
    private Date startDate;
    private Date endDate;
}
